app/v2/notifications/post_notifications.py

```diff
@@ -428,22 +428,25 @@ def process_sms_or_email_notification(
 
     personalisation = process_document_uploads(form.get("personalisation"), service, simulated, template.id)
 
-    _notification: NotificationDictToSign = {
-        "id": create_uuid(),
-        "template": str(template.id),
-        "service_id": str(service.id),
-        "template_version": str(template.version),  # type: ignore
-        "to": form_send_to,
-        "personalisation": personalisation,
-        "simulated": simulated,
-        "api_key": str(api_key.id),
-        "key_type": str(api_key.key_type),
-        "client_reference": form.get("reference", None),
-        "reply_to_text": reply_to_text,
-    }
+    _notification = PendingNotification.from_dict(
+        {
+            "id": create_uuid(),
+            "template": str(template.id),
+            "service_id": str(service.id),
+            "template_version": str(template.version),  # type: ignore
+            "to": form_send_to,
+            "personalisation": personalisation,
+            "simulated": simulated,
+            "api_key": str(api_key.id),
+            "key_type": str(api_key.key_type),
+            "client_reference": form.get("reference", None),
+            "reply_to_text": reply_to_text,
+            # "queue": None,  # review if necessary
+            # "sender_id": None,  # review if necessary
+        }
+    )
 
-    signed_notification_data = signer_notification.sign(_notification)
```

Comment:
This layer does not signed the notification anymore: we want to delay the signing in the Redis queue because we need to add extra metadata in there, some coupled to queue management. Hence we removed the signing and Redis will do it itself, via the new signing annotations.